dev/lint-python

```diff
@@ -82,6 +87,34 @@ else
     rm "$PYCODESTYLE_REPORT_PATH"
 fi
 
+# Check python document style, skip check if pydocstyle is not installed.
+if hash "$PYDOCSTYLEBUILD" 2> /dev/null; then
+    if [ $PYDOCSTYLEVERSION == "2.1.1" ]; then
```

Comment:
What happens when a new version of pydocstyle is released?  Would the same rules below still work so that this check could be `>= 2.1.1`?

Reply:
done